e2e-tests/src/accounts.rs

```diff
@@ -5,26 +5,29 @@ use substrate_api_client::Balance;
 use crate::config::Config;
 use crate::{Connection, KeyPair};
 
-pub fn keypair_from_string(seed: String) -> KeyPair {
-    KeyPair::from_string(&seed, None).expect("Can't create pair from seed value")
+pub fn keypair_from_string(seed: &str) -> KeyPair {
+    KeyPair::from_string(seed, None).expect("Can't create pair from seed value")
 }
 
-pub fn accounts_from_seeds(seeds: Option<Vec<String>>) -> Vec<KeyPair> {
-    let seeds = seeds.unwrap_or_else(|| {
-        vec![
-            "//Damian".into(),
-            "//Tomasz".into(),
-            "//Zbyszko".into(),
-            "//Hansu".into(),
-        ]
-    });
-    seeds.into_iter().map(keypair_from_string).collect()
+pub fn accounts_from_seeds(seeds: Option<&Vec<String>>) -> Vec<KeyPair> {
+    if let Some(seeds) = seeds {
+        seeds
+            .iter()
+            .map(String::as_str)
+            .map(keypair_from_string)
+            .collect()
+    } else {
+        ["//Damian", "//Hansu", "//Tomasz", "//Zbyszko"]
```

Comment:
Good point :)

Reply:
Ah, you can make `default_accounts` return `Vec<KeyPair>` directly, and then again the code below becomes even simpler.

Reply:
Then `default_account_seeds` is a slightly better name imo.

Also (related to the comment below that I resolved and don't want to risk unresolving, 'cause RAM), I think you could make this function take an option of a vector, rather than a reference to a vector, since afaict in all uses you have to go out of your way to change a vector into a reference.